### PR TITLE
Issue #14436: Enforced naming convention for illegalTokenTextCheckTest

### PR DESCRIPTION
**Pull request by Pinti0001**
Part of Issue no. #14436 

Enforced naming convention on illegalTokenTextCheckTest

### CODE REVIEW

**Comment by Pinti0001** [comment]
@romani 
Is the naming accurate ?

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokensCheckCommentToken.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 /**
  * Test for illegal tokens
  */
-public class InputIllegalTokenTextTokens5
+public class InputIllegalTokenTextTokensCheckCommentToken
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokensCheckCommentToken.java:16]: InputIllegalTokenTextTokensCheckCommentToken --> `InputIllegalTokenTextCheckCommentToken`

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokensCheckCustomMessage.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
 /**
  * Test for illegal tokens
  */
-public class InputIllegalTokenTextTokens3
+public class InputIllegalTokenTextTokensCheckCustomMessage
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokensCheckCustomMessage.java:16]: InputIllegalTokenTextTokensCheckCustomMessage --> `InputIllegalTokenTextCheckCustomMessageInStringLiteral`

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokensCheckNullCustomMessage.java [hunk 3]
```diff
@@ -13,7 +13,7 @@
 /**
  * Test for illegal tokens
  */
-public class InputIllegalTokenTextTokens4
+public class InputIllegalTokenTextTokensCheckNullCustomMessage
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokensCheckNullCustomMessage.java:16]: InputIllegalTokenTextTokensCheckNullCustomMessage --> `InputIllegalTokenTextCheckNullMessageInStringLiteral`

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokensCheckTestCaseInSensitive.java [hunk 4]
```diff
@@ -13,7 +13,7 @@
 /**
  * Test for illegal tokens
  */
-public class InputIllegalTokenTextTokens2
+public class InputIllegalTokenTextTokensCheckTestCaseInSensitive
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokensCheckTestCaseInSensitive.java:16]: InputIllegalTokenTextTokensCheckTestCaseInSensitive --> `InputIllegalTokenTextCheckCaseInSensitive`

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokensCheckTestCaseSensitive.java [hunk 5]
```diff
@@ -13,7 +13,7 @@
 /**
  * Test for illegal tokens
  */
-public class InputIllegalTokenTextTokens
+public class InputIllegalTokenTextTokensCheckTestCaseSensitive
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokensCheckTestCaseSensitive.java:16]: InputIllegalTokenTextTokensCheckTestCaseSensitive --> `InputIllegalTokenTextCheckDefaultCaseSensitive`

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheckTest.java [hunk 6]
```diff
@@ -42,46 +42,46 @@ protected String getPackageLocation() {
     }
 
     @Test
-    public void testCaseSensitive()
+    public void testIllegalTokenTextTokensCheckTestCaseSensitive()
```
Comment (MANISH-K-07) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheckTest.java:45]: testIllegalTokenText--`Tokens`--CheckTestCaseSensitive

Please remove this word from all such occurences and also update the test method names according to new input filenames that I have suggested as part of review

**Review by MANISH-K-07 — COMMENTED** [review]
```
Commit messages from github: "Issue #[14](https://github.com/checkstyle/checkstyle/actions/runs/8127713098/job/22212843269?pr=14592#step:3:15)4436: Enforced naming convention for illegaltokent...
Issue number: #144436
PR Description grepped:
Please put a reference to an Issue in the PR description,
this will bind the Issue to your PR in Github
jq: error (at <stdin>:4): Cannot iterate over null (null)
You are providing a PR for an Issue that is not approved yet,
please ask admins to approve your Issue first
PR validation failed.
```
Please update your PR name to `Issue #14436: ......`

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheckTest.java (hunk 6 above)
Comment (Pinti0001) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheckTest.java:45]: ok @MANISH-K-07 got it

Comment (Pinti0001) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheckTest.java]: @MANISH-K-07  see what is the problem now

**Review by MANISH-K-07 — COMMENTED** [review]
The changes lgtm....

Failing CI are due to 2 commits.
Please squash and rebase to a single commit.

**Comment by Pinti0001** [comment]
> The changes lgtm....
> 
> Failing CI are due to 2 commits. Please squash and rebase to a single commit.

ok got it

**Comment by Pinti0001** [comment]
Hey @romani @MANISH-K-07  Please check now

**Review by MANISH-K-07 — COMMENTED** [review]
There is still a failing CI
Checkstyle needs your PR to be limited to a single commit. Please squash and rebase to latest commit to make CI green

**Pull request closed without merging**